Weather/Services/UserLocations/UserLocationsService.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+
+protocol UserLocationsService {
+    
+    func storeLocation(location: WeatherLocation)
```

Comment:
can we call it 

addLocation

as so is more clear that we are adding a new location?